DuckDB.NET/MacOS/NativeMethods.MacOS.cs

```diff
@@ -189,7 +189,7 @@ public void DuckDBFree(IntPtr ptr)
 
     public class NativeMethods
     {
-        private const string DuckDbLibrary = "libduckdb.dylib";
+        private const string DuckDbLibrary = "duckdb.dylib";
```

Comment:
The precompiled binary for Mac is called `libduckdb.dylib`

Reply:
I can only speak to what I saw on CI as I don't own a Mac. The CI run (on MacOS) failed because it was searching for liblibduckdb.dylib for some reason (.NET might automatically prepend the lib?).

But, yes the precompiled binary for Mac is called libduckdb.dylib.

Reply:
According to [Writing cross platform P/Invoke code](https://docs.microsoft.com/en-us/dotnet/standard/native-interop/cross-platform#library-name-variations) it will try with and without lib prefix

Reply:
Actually this got me thinking: if the extension is appended at runtime depending on the platform, can we just have one class for binding instead of three?

Reply:
Yes, I think so. Then I guess, this:

```
         private const string DuckDbLibrary = "duckdb.(dll|so|dylib)";
```

should just be:
```
         private const string DuckDbLibrary = "duckdb";
```

in the one and only bindings class.

Reply:
The question is whether or not it will find duckdb.dll when using the library from .Net Framework project.